controllers/reviews/deleteReviewById/deleteReviewById.js

```diff
@@ -0,0 +1,26 @@
+import { HttpError } from '../../../helpers/HttpError.js';
+import { isValidId } from '../../../middlewares/isValidId.js';
+import { Review } from '../../../models/review/index.js';
+
+export const deleteReviewById = async (app) => {
+  app.delete(
+    '/reviews/:reviewId',
+    // add authenticate middleware
+    // authenticate,
+
+    isValidId,
+    async (req, res, next) => {
+      try {
+        const { reviewId } = req.params;
+        const result = await Review.findByIdAndDelete(reviewId).exec();
+
+        if (!result) {
+          throw HttpError(404, 'NotFound');
```

Comment:
I don't think we need error here
as a result item which satisfies search params should not be stored in DB, if it's not found there - it fits our goal, item is not stored

we can log this info instead, to be aware, with data about search query and user

Reply:
Could you please explain me, why we haven't send error from here?

If there isn't a review in db  I throw Error 404 Not found, and this error will be caught in our last middleware.

About log I will discuss with team how we logging errors.

Reply:
I'm not sure if deleting already non-existent value should be treated as an error, Mongoose doesn't consider it as an error, for example, and returns just `null` instead of meta object

your approach isn't wrong as well, so it will be just something consider in a future
leave it as it is now